bertinNDAYISHIMIYE/capstoneProject-MyBrand
# ft-create-blog

###What does this PR do?

>add create blog routes
>add blog validation middleware
>add create blog logic into controllers

###Description of the task to be completed!

>Check if the user is authorised
>User to create a blog
>Deny creation of blog with invalid data

###How should this be tested manually?

>Clone this repo with: https://github.com/bertinNDAYISHIMIYE/capstoneProject-MyBrand
>Checkout to ft-createBlog branch
>Make sure node.js is installed
>Run command npm init for npm initialisation
>Run npm i to install different packages
>Use postman as Third party software to test the end point

###Any background context you want to provide?

>N/P
###Relavant trello card
https://trello.com/c/pvLMoTVW

###Screenshot
<img width="1416" alt="Screen Shot 2020-12-09 at 20 17 11" src="https://user-images.githubusercontent.com/72761713/101670219-90c1ff80-3a5b-11eb-89de-9f8579d3aa83.png">
